node/node.go

```diff
@@ -700,10 +700,8 @@ func (n *nodeImpl) OnStart() error {
 	}
 
 	if n.config.Mode != config.ModeSeed {
-		if n.config.BlockSync.Version == config.BlockSyncV0 {
```

Comment:
Maybe at this point we should just remove this setting from the config file entirely? At least at master and maybe in v0.36 that seems safe.

Reply:
I believe it's out in master and 0.36 this is just against 0.35